apps/balancer-tools/src/app/poolsimulator/(components)/PoolParamsForm.tsx

```diff
@@ -169,6 +182,18 @@ export function PoolParamsForm() {
             </span>
           </div>
         ))}
+        <div className="flex flex-col">
+          <label className="mb-2 block text-sm text-slate12">Tokens</label>
+          {errors?.tokens?.message && (
```

Comment:
👍  good strategy

Reply:
No, this is from [PYC's PR](https://github.com/bleu-studio/balancer-tools/pull/170)